utilities/test_suite/HIP/runAudioTests.py

```diff
@@ -237,6 +234,7 @@ def rpp_test_suite_parser_and_validator():
             srcPath = inFilePath
 
     if case not in supportedCaseList:
+        print("\nCase " + case + " is not supported.")
```

Comment:
Adding this print statement here again, prints a lot of statements for which cases are not supported
We need to add a print statement **only if the none of the cases requested by user is supported** 

if noCaseSupported:
 print "case numbers {case_list} is not supported"

Reply:
Done